packages/vite/src/node/utils.ts

```diff
@@ -38,8 +38,10 @@ export const normalizeId = (id: string): string =>
   id.replace(/(\s*>\s*)/g, ' > ')
 
 export function isBuiltin(id: string): boolean {
-  const deepMatch = id.match(deepImportRE)
-  id = deepMatch ? deepMatch[1] || deepMatch[2] : id
+  const builtins = [...builtinModules]
```

Comment:
Should we use a `Set`? Are duplicates relevant anyways?
This or that way, we could avoid the if branches 🙂

Reply:
A set is a good idea, @Niputi lets move the generation of the list to a constant outside of isBultin so we dont regenerate it each time